ubuntudesign/vanilla-framework
# Use example.js to display the examples in the docs

## Done
Replaced the hard-coded examples in the docs with a example.js link

## QA
- Run `npm link` in your vanilla-framework 
- Then run `gulp develop`
- Jump over to docs.vanillaframework.io branch
- Pull this branch
- Run `npm link vanilla-framework`
- Run `gulp develop` 
- Check through the pages and see that the examples (although not pretty) work and display the pattern with the code below it

## Details
Fixes https://github.com/ubuntudesign/vanilla-framework/issues/801
